pkg/asset/machines/worker.go

```diff
@@ -509,6 +535,7 @@ func (w *Worker) MachineSets() ([]machineapi.MachineSet, error) {
 		ovirtprovider.SchemeGroupVersion,
 		vsphereprovider.SchemeGroupVersion,
 		kubevirtprovider.SchemeGroupVersion,
+		powervsprovider.GroupVersion, //TODO: Should we rename from GroupVersion to SchemeGroupVersion
```

Comment:
can you link me to to more info on that one?

Reply:
This has to be changed in power vs provider, Currently we are using varibale name as GroupVersion(https://github.com/openshift-powervs/cluster-api-provider-powervs/blob/61016a28718618344d36ff8a4f81d5b27a8bace4/pkg/apis/powervsprovider/v1alpha1/groupversion_info.go#L35)
All the other providers naming it as SchemeGroupVersion
For example for kubervirt: https://github.com/openshift/cluster-api-provider-kubevirt/blob/b0e15d7907de5bfbf4c64edff01bd67571b7ae75/pkg/apis/kubevirtprovider/v1alpha1/register.go#L32

Reply:
ah, yah. it looks like the upstream/kube providers use `GroupVersion`, but the OCP providers seem to use `SchemaGroupVersion`. You can either check with @mkumatag or go ahead and submit a PR to our provider repo to change it there if you want, and he can comment there :D

Reply:
@Karthik-K-N I don't mind accepting the changes into the `cluster-api-provider-powervs` if no side effect later by [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) utility